server/src/test/java/org/elasticsearch/action/bulk/TransportShardBulkActionTests.java

```diff
@@ -64,9 +64,9 @@
 import static org.hamcrest.Matchers.arrayWithSize;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.instanceOf;
-import static org.mockito.Mockito.any;
-import static org.mockito.Mockito.anyBoolean;
-import static org.mockito.Mockito.anyLong;
+import static org.mockito.Matchers.any;
```

Comment:
`Mockito.whatever` is more idiomatic usage for mockito.

Reply:
Good to know, will try it some time. Was wondering how this line snuck into this PR since I don't remember having changed anything around the mocks, but apparently my IDE was just resorting the imports.